src/typescript-service.ts

```diff
@@ -1619,7 +1632,12 @@ export class TypeScriptService {
         // Ensure files needed to suggest completions are fetched
         await this.projectManager.ensureReferencedFiles(uri).toPromise()
 
+        const config = this.projectManager.getParentConfiguration(uri)
```

Comment:
Consider moving the new code here into `ProjectManager.didClose`.

You're changing the behavior here by skipping `this.projectManager.didClose` if no config is found.